src/front/shared/pages/Exchange/Exchange.tsx

```diff
@@ -191,33 +182,81 @@ const bannedPeers = {} // rejected swap peers
       nextData,
       ...Object.values(tokensData).filter(({ address }) => address),
       ...Object.values(rest)
-        //@ts-ignore
         .filter((coinData) => coinData && coinData.address)
         .filter(({ address }) => address),
     ],
   })
 )
 @CSSModules(styles, { allowMultiple: true })
 export default class Exchange extends Component<any, any> {
-  
+  /**
+   * @method getDefaultWalletForCurrency
+   * @method getExchangeDataFromLocalStorage
+   * @method getLinkToDeclineSwap
+   * @method getUserDefaultWallet
+   * @method getCorrectDecline
+   * @method getCoinFullName
+   * @method getFiatBalance
+   * @method getCoinData
+   * @method getBalance
+   *
+   * @method setUserDefaultWallet
+   * @method setAmountOnState
+   * @method setDeclinedOffer
+   * @method setOrderOnState
+   * @method setNoOfferState
+   * @method setAmount
+   * @method setOrders
+   *
+   * @method checkBalanceOnAllCurrency
+   * @method checkoutLowAmount
+   * @method checkSwapAllow
+   * @method checkValidUrl
+   * @method checkUrl
+   * @method checkPair
+   *
+   * @method makeAddressObject
+   * @method rmScrollAdvice
+   * @method changeUrl
+   * @method fetchPairFeesAndBalances
+   * @method fetchFiatExRate
+   * @method createOffer
+   * @method initSwap
+   * @method openModalDeclineOrders
+   * @method sendRequestForPartial
+   * @method returnNeedCurrency
+   * @method findGoodOrder
+   * @method isPeerBanned
+   * @method banPeer
+   * @method handleSetGetValue
+   * @method handleSetHaveValue
+   * @method applyAddress
+   * @method flipCurrency
+   * @method resetState
+   * @method chooseCurrencyToRender
+   * @method extendedControlsSet
+   * @method doesComissionPreventThisOrder
+   * @method goDeclimeFaq
+   * @method showIncompleteSwap
+   */
```

Comment:
```suggestion
```

Generated? JSDoc seems not very helpful.

Reply:
For large files, it's better when all methods in one place

Reply:
agree, will be better make docs for more important code